### ISSUE TITLE
Flsimulate : primary_event:versatile_generator not working on Falaise 3

### ISSUE THREAD

**Issue by noeroy** [SuperNEMO-DBD/Falaise#11]
Hello,
I tried to simulate gammas with 5MeV energy with the primary_event : versatile_generator. And it simulated 1MeV gammas. I also tried 100keV electrons and it simulated 1MeV gammas (the default value). I used a `variants.profile`, here it is : 

```
#@format=datatools::configuration::variant
#@format.version=1.0
#@organization=snemo
#@application=falaise

[registry="geometry"]
layout = "Basic"
layout/if_basic/magnetic_field = true
layout/if_basic/magnetic_field/is_active/type = "UniformVertical"
layout/if_basic/magnetic_field/is_active/type/if_uniform_vertical/magnitude = 25 gauss
layout/if_basic/magnetic_field/is_active/type/if_uniform_vertical/direction = "+z"
layout/if_basic/source_layout = "Basic"
layout/if_basic/source_layout/if_basic/thickness = 250 um
layout/if_basic/source_layout/if_basic/material = "Se82"
layout/if_basic/source_calibration = false
layout/if_basic/shielding = true
calo_film_thickness = 25 um

[registry="vertexes"]
generator = "calo_8inch_back_scin_bulk"

[registry="primary_events"]
generator = "versatile_generator"
generator/if_versatile/particle = "gamma"
generator/if_versatile/energy = 5000 keV

[registry="simulation"]
physics_mode = "Constructors"
physics_mode/if_constructors/em_model = "standard"
production_cuts = true
output_profile = "none"

```

and here my `.conf` 
```

#@description Main flsimulate configuration script
#@key_label  "name"
#@meta_label "type"

[name="flsimulate" type="flsimulate::section"]
numberOfEvents : integer = 1000

[name="flsimulate.variantService" type="flsimulate::section"]
profile : string as path = "/unix/nemo2/users/noeroy/Gammas/variants.profile"

```

**Comment by drbenmorgan** [comment]
Just to cross-reference, this is being addressed in PR #20

**Comment by goliviero** [comment]
Same issue as : https://github.com/SuperNEMO-DBD/Falaise/issues/21
It was fixed in the PR https://github.com/SuperNEMO-DBD/Falaise/pull/20
